README.md

```diff
@@ -41,13 +41,15 @@ use Kaldi as an example)
 git clone https://github.com/SpeechColab/GigaSpeech.git
 
 cd GigaSpeech
-toolkits/kaldi/run_gigaspeech.sh data/
+toolkits/kaldi/gigaspeech_download.sh
```

Comment:
@wangyongqing0731 the script is BROKEN here.  After clone, GIGASPEECH_LOCAL_ROOT is not adapt to their local machine, if we organize the kaldi script this way, we should at least provide options to set GIGASPEECH_LOCAL_ROOT, otherwise what users get from this script is ERROR.

Reply:
Removed GIGA_SPEECH_LOCAL_ROOT